gulp/i18n.js

```diff
@@ -77,3 +79,16 @@ gulp.task(
             .pipe(gulp.dest('./'))
     )
 );
+
+gulp.task(
+    'pull-blockly-translations',
+    gulp.series(done => {
+        const blocklyLanguages = ['en', 'de', 'id', 'it', 'vi', 'pl', 'ru', 'pt', 'es', 'fr', 'zh-hans', 'zh-hant'];
+        remoteSrc(blocklyLanguages.map(lang => `${lang}.js?_=${Date.now()}`), {
+            base: 'https://blockly-demo.appspot.com/static/msg/js/',
+        })
+            .pipe(rename(path => (path.extname = '.js')))
+            .pipe(gulp.dest('www/translations'));
```

Comment:
i thought it might be `www/translations/`

Reply:
It's consistent with the rest of the build scripts `gulp.dest` calls
- https://gulpjs.org/API.html#gulp-dest-path-options